packages/account-api/src/api/provider.ts

```diff
@@ -33,18 +43,15 @@ export type AccountProvider<Account extends KeyringAccount> = {
   }) => Promise<Account[]>;
 
   /**
-   * Discover accounts for a given entropy source and a given group
-   * index.
+   * Discover accounts for a given entropy source.
    *
    * NOTE: This method needs to also create the discovered accounts.
    *
    * @param options - Options.
    * @param options.entropySource - Entropy source to use.
-   * @param options.groupIndex - Group index to use.
    * @returns The list of discovered and created accounts.
    */
   discoverAndCreateAccounts: (options: {
     entropySource: EntropySourceId;
-    groupIndex: number;
```

Comment:
Will the index be managed inside the provider now?

Reply:
yessir.